src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -228,10 +229,10 @@ int SQLPTRepresentation::TimeoutResponse() {
   utils::dbms::SQLQuery query(db());
   if (!query.Prepare(sql_pt::kSelectTimeoutResponse) || !query.Exec()) {
     LOG4CXX_INFO(logger_, "Can not select timeout response for retry sequence");
-    const int kDefault = 30;
+    const int kDefault = 30000;
```

Comment:
The meaning of the value will be more clear if the const variable is initialized as 30 * date_time::DateTime::MILLISECONDS_IN_SECOND